app/templates/blog/update.html

```diff
@@ -0,0 +1,40 @@
+{% extends "blog_main.html" %}
+
+{% block blogContent %}
+<div class="column">
+    <div class="box">
+        <form method="POST" action="/blog/{{ post.id }}/update">
+            <div class="mb-3">
+                <label for="title" class="form-label">Title</label>
+                <input
+                class="form-control"
+                type="text"
+                name="title"
+                id="title"
+                placeholder="{{ post.title }}"
+                value="{{ post.title }}">
```

Comment:
We don't need to leave things like this. I was getting dizzy scrolling to the right!